content/docs/composition-vs-inheritance.md

```diff
@@ -28,7 +28,8 @@ function FancyBorder(props) {
 }
 ```
 
-This lets other components pass arbitrary children to them by nesting the JSX:
+Acest lucru permite altor componente să introducă in ele copii la întâmplare prin cuibărind JSX-ul:
```

Comment:
Instead of 'cuibarind' I would use a longer translation. Something like: 'prin adaugarea codului JSX intre tag-urile componentei'.